handler.go

```diff
@@ -55,9 +63,22 @@ func NewWASMServer(wasmFile string, args []string, coverageFile string, l *log.L
 		srv.envMap[vars[0]] = vars[1]
 	}
 
-	buf, err := os.ReadFile(path.Join(runtime.GOROOT(), "misc/wasm/wasm_exec.js"))
-	if err != nil {
-		return nil, err
+	var buf []byte
+	for _, loc := range wasmLocations {
+		buf, err = os.ReadFile(filepath.Join(runtime.GOROOT(), loc))
+		if err == nil {
+			break
+		}
+		if !os.IsNotExist(err) {
+			return nil, err
+		}
+	}
+	if len(buf) == 0 {
```

Comment:
This will anyways be unnecessary after 1.24, and given that it's possible to copy over the file, I'd prefer to avoid bundling the file with the tool. Let's just use the 2 locations to check for the the file, and on failure, give a clear warning for users to either use 1.24 or copy the file in that location.